library/core/src/num/int_macros.rs

```diff
@@ -1179,7 +1203,7 @@ macro_rules! int_impl {
         #[rustc_const_stable(feature = "const_checked_int_methods", since = "1.47.0")]
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
-        #[inline]
+        #[inline(always)]
```

Comment:
FYI, functions which are `inline(always)` are inlined even in unoptimized builds. rustc takes extra steps to make this happen, and I disagree very strongly with that behavior, but I'm bringing this up because this may explain some surprising impacts of this PR on debug build times. Don't know yet.

My PR to stop this behavior of `inline(always)`: https://github.com/rust-lang/rust/pull/121417#issuecomment-1958597456

Reply:
So, I initially added this after the stuff I mentioned [here](https://github.com/rust-lang/rust/pull/117494#issuecomment-1960646918), but I did just leave it based upon assumptions that are probably wrong, so, I can remove it.